resources/lib/sutils.py

```diff
@@ -173,12 +177,14 @@ def add_item(self, params):
                 sub['name']), self.normalize_filename(params['name']) + '.nfo')
             if not xbmcvfs.exists(nfo_file):
                 metadata = ""
-                if 'imdb' in params:
+                if ('imdb' in params and params['imdb'] is not None and
+                        re.match('^$|^[?0]$', params['imdb']) is None):
```

Comment:
Nepotrebna zatvorka, `is not None` nie je potrebne pisat a `re.match('^$|^[?0]$', params['imdb']) is None` vies nahradit za `not re.match('^$|^[?0]$', params['imdb'])`.